test/FsAutoComplete.IntegrationTests/DotNetCore/notrestored.json

```diff
@@ -1,14 +0,0 @@
-{
```

Comment:
not restored test has been really problematic for a while, I'd be OK with just removing it